### PR TITLE
Lint yml test, and lint fixes

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Lint github action yml files for validity, best practices.
Uses https://github.com/adrienverge/yamllint
Documentation: https://yamllint.readthedocs.io/en/stable/index.html
I would encourage real review and discussion/clean up on this...

### CODE REVIEW

File: .github/workflows/commit.yml [hunk 1]
```diff
@@ -93,7 +103,9 @@ jobs:
       - name: deploy ${{ matrix.app }}
         uses: cloud-gov/cg-cli-tools@main
         with:
-          command: cf push ${{ matrix.app }} --vars-file vars.development.yml --strategy rolling
+          command: >
+            cf push ${{ matrix.app }} --vars-file \
+            vars.development.yml --strategy rolling
```
Comment (robert-bryson) [inline .github/workflows/commit.yml:108]: It's very cool that it does this for us.

Comment (jbrown-xentity) [reply .github/workflows/commit.yml]: Actually good catch, I didn't need the `\` there, command works differently. See previously working version [here](https://github.com/GSA/catalog.data.gov/blob/main/.github/workflows/db-solr-sync.yml#L40-L43)

Comment (robert-bryson) [reply .github/workflows/commit.yml]: I believe the behavior of `>` in yaml can let us break some of the other long commands into their own lines too, like in those couple that you had to disable the length linting. `>` should just append each following line without adding a newline char, right?

Comment (jbrown-xentity) [reply .github/workflows/commit.yml]: Possibly. Didn't have time to look into the details, just using it for what I know would work (in the cloud.gov start command field)...

**Review by robert-bryson — DISMISSED** [review]

**Review by robert-bryson — APPROVED** [review]

**Comment by jbrown-xentity** [comment]
Looks much better to me, thanks for the assist @robert-bryson!

**Pull request merged**